diff --git a/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/index.js b/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/index.js
index 6278734..ccc4dec 100644
--- a/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/index.js
+++ b/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/index.js
@@ -13,6 +13,8 @@ const richTextEditor = ({
   defaultValue = null,
   disabledButtons = [],
   hideOnMobileButtons = [],
+  presetColors = [],
+  compact = false,
   website,
   ...config
 } = {}) => meta => {
@@ -56,7 +58,9 @@ const richTextEditor = ({
     getAdminMeta: () => ({
       defaultValue,
       disabledButtons,
-      hideOnMobileButtons
+      hideOnMobileButtons,
+      presetColors,
+      compact
     })
   }, {
     default: defaultValue === null ? undefined : {
diff --git a/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/views/utils.js b/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/views/utils.js
index eaa5425..f25dc57 100644
--- a/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/views/utils.js
+++ b/node_modules/@mirrormedia/lilith-core/lib/custom-fields/rich-text-editor/views/utils.js
@@ -22,11 +22,13 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 
 function createController(decorators) {
   return config => {
-    var _config$fieldMeta, _config$fieldMeta2;
+    var _config$fieldMeta, _config$fieldMeta2, _config$fieldMeta3;
 
     return {
       disabledButtons: ((_config$fieldMeta = config.fieldMeta) === null || _config$fieldMeta === void 0 ? void 0 : _config$fieldMeta.disabledButtons) ?? [],
       hideOnMobileButtons: ((_config$fieldMeta2 = config.fieldMeta) === null || _config$fieldMeta2 === void 0 ? void 0 : _config$fieldMeta2.hideOnMobileButtons) ?? [],
+      presetColors: ((_config$fieldMeta3 = config.fieldMeta) === null || _config$fieldMeta3 === void 0 ? void 0 : _config$fieldMeta3.presetColors) ?? [],
+      compact: ((_config$fieldMeta3 = config.fieldMeta) === null || _config$fieldMeta3 === void 0 ? void 0 : _config$fieldMeta3.compact) ?? false,
       path: config.path,
       label: config.label,
       description: config.description,
@@ -84,6 +86,8 @@ function createField(RichTextEditor) {
     return /*#__PURE__*/_react.default.createElement(_fields.FieldContainer, null, /*#__PURE__*/_react.default.createElement(_fields.FieldLabel, null, field.label, /*#__PURE__*/_react.default.createElement(_core.Stack, null, /*#__PURE__*/_react.default.createElement(RichTextEditor, {
       disabledButtons: field.disabledButtons,
       hideOnMobileButtons: field.hideOnMobileButtons,
+      presetColors: field.presetColors,
+      compact: field.compact,
       editorState: value,
       onChange: // @ts-ignore: any
       editorState => onChange === null || onChange === void 0 ? void 0 : onChange(editorState)
